### ISSUE TITLE
get_data_frame bug with multiple receivers

### ISSUE THREAD

**Issue by lilyclements** [IDEMSInternational/R-Instat#3665]
I noticed this in dlgDescribeTwoVarGraph. It is an issue in other graphics dialogs if you use the ucrFactorsToVariates as a multiple receiver.

"First Variables" isn't clearing on Two Variable Graph when switching data sets in `id.vars`, it keeps the old variable(s) and adds the new to the list of variables in the string. It is in `measure.vars`
This is a problem in `get_data_frame` I believe
So, I ran this initially with Survey, and then with Dodoma. When running it with Dodoma I used the variables "Year" and "Rain". (When with survey, I used something and "Fert")

It tried to run:

`dodoma_temp <- InstatDataObject$get_data_frame(stack_data=TRUE, id.vars=c("Fert","Year"), data_name="dodoma", measure.vars="Rain")`

which caused an error since "Fert" isn't in "dodoma".

**Comment by AlexSananka** [comment]
How did you achieve this?I thought the second receiver is always a single receiver

**Comment by lilyclements** [comment]
Yes so the second is always single. This error was occurring with the first receiver which is multiple.

I could also get this to occur on the plot dialogs if I used the multiple receiver not single receiver.

**Comment by dannyparsons** [comment]
This is fixed for the case described above with two data frames. Are there any remaining related issues?

**Comment by maxwellfundi** [comment]
@lilyclements  please re-open this because we experienced a similar error. 

So the id vars, when  having two dataframes does not update appropriately when data frames are changed and hence we get graphs with id vars of the previous dataframe

**Comment by lilyclements** [comment]
It won't let me reopen? Perhaps because I didn't close it? @dannyparsons would you mind reopening this?

**Comment by dannyparsons** [comment]
Still kind replicate this, will leave open but move to an un-dated milestone.
